predtimechart-config.yml

```
@@ -5,4 +5,4 @@ reference_date_col_name: 'reference_date'
 target_date_col_name: 'target_end_date'
 horizon_col_name: 'horizon'
 initial_checked_models: ['UMass-flusion', 'UMass-trends_ensemble']
-disclaimer: Most forecasts have failed to reliably predict rapid changes in the trends of reported cases and hospitalizations. Due to this limitation, they should not be relied upon for decisions about the possibility or timing of rapid changes in trends.
+disclaimer: 
```

Use explicit YAML null indicator instead of leaving it blank. 

```suggestion
disclaimer: ~
```